CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## UNRELEASED
 
+BUG FIXES:
+* Helm
+  * update client-daemonset to not include ca-cert volumeMount with tls disabled [[GH-1194](https://github.com/hashicorp/consul-k8s/pull/1194)]
```

Comment:
Non-blocking but the double negative is getting me. Maybe:

`update client-daemonset to only include ca-cert volumeMount when tls is enabled`

Reply:
```suggestion
  * update client-daemonset to include ca-cert volumeMount when tls is enabled [[GH-1194](https://github.com/hashicorp/consul-k8s/pull/1194)]
```